config/sync/field.storage.node.layout_builder__layout.yml

```diff
@@ -1,4 +1,4 @@
-uuid: 833a0790-a592-43cb-90d6-f15509b9c74a
+uuid: ed8c3733-5cf1-4e6d-940e-350e7c8d76b8
```

Comment:
We cant change the uuid of existing config

Reply:
Good catch. I'll look for more of these.